msc_pygeoapi/provider/hrdps_zarr.py

```diff
@@ -38,7 +38,6 @@
     ProviderInvalidQueryError,
     ProviderNoDataError
 )
-import dask.array as da
```

Comment:
If dask is removed from this provider, then remove dask from `requirements.txt` as well.